WebPhapp/client/src/App/components/Prescription.js

```diff
@@ -268,7 +271,19 @@ class Prescription extends Component {
                         </div>
                         </div>
                         </div>
-                      }
+                    }
+                    {
+                    <div className="modal fade-up" id="cancel-prescription-modal" tabindex="-1" role="dialog" aria-labelledby="modal-default" aria-hidden="true">
+                        <div className="modal-dialog modal-dialog-bottom modal-" role="document">
+                            <div className="alert alert-danger alert-dismissible" role="alert">
+                                <span className="alert-inner--text"><strong> CANCELLED: </strong> Prescription cancelled from Pharmachain.</span>
```

Comment:
Maybe do some more view on the rendering of this? I have had issues with the location of this pop up for the cancel. @andrewjyang